app/ios/STA.xcodeproj/project.pbxproj

```diff
@@ -994,7 +994,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Also not sure these changes should be included in this PR.  If they're general changes that do need to be made for the iOS setup, I think that should go in a separate PR (and so it can be flagged for testing by other Mac users, etc).